### PR TITLE
1926 Rename DataSyncSyncer to MeasurementsSyncer

### PR DESCRIPTION
**Pull request by tnickelsen**
and now uses measurements instead of datasync. Auth now uses DataHubFacade instead of DataSync.API

### CODE REVIEW

File: domains/auth/Auth.API/Integration.Tests/Extensions/NSubstituteExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+using Grpc.Core;
+using Grpc.Core.Testing;
+using NSubstitute;
+using NSubstitute.Core;
+
+namespace Integration.Tests.Extensions;
```
Comment (TopSwagCode) [inline domains/auth/Auth.API/Integration.Tests/Extensions/NSubstituteExtensions.cs:6]: Move this to:
<ProjectReference Include="..\..\Shared\Testing\Testing.csproj" />

Shared?

File: domains/auth/Auth.API/Unit.Tests/Extensions/NSubstituteExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+using Grpc.Core;
+using Grpc.Core.Testing;
+using NSubstitute.Core;
+
+namespace Unit.Tests.Extensions;
+
+public static class NSubstituteExtensions
+{
```
Comment (TopSwagCode) [inline domains/auth/Auth.API/Unit.Tests/Extensions/NSubstituteExtensions.cs:8]: Move to 
<ProjectReference Include="..\..\Shared\Testing\Testing.csproj" />
?

File: domains/auth/Auth.API/Integration.Tests/Extensions/NSubstituteExtensions.cs (hunk 1 above)
Comment (tnickelsen) [reply domains/auth/Auth.API/Integration.Tests/Extensions/NSubstituteExtensions.cs:6]: I felt it was a bit much to make a new project just for this. But I see your point - want me to move it to shared project?

File: domains/auth/Auth.API/Integration.Tests/AuthWebApplicationFactory.cs [hunk 3]
```diff
@@ -39,6 +39,8 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
     {
         builder.UseEnvironment("Test");
 
+        builder.UseSetting("DataHubFacade:Url", "http://foo");
```
[the remark this answers is not in the document]
Comment (tnickelsen) [reply domains/auth/Auth.API/Integration.Tests/AuthWebApplicationFactory.cs]: Done

**Review by sahma19 — DISMISSED** [review]

File: domains/auth/Auth.API/Integration.Tests/Extensions/NSubstituteExtensions.cs (hunk 1 above)
Comment (tnickelsen) [reply domains/auth/Auth.API/Integration.Tests/Extensions/NSubstituteExtensions.cs]: It's been moved to Testing project

**Review by TopSwagCode — APPROVED** [review]

**Pull request merged**